packages/vue-cli-plugin-vuetify/generator/tools/vuetify.js

```diff
@@ -4,7 +4,7 @@ const helpers = require('./helpers')
 function addDependencies (api, v3) {
   api.extendPackage({
     dependencies: {
-      vuetify: v3 ? '^3.0.0-alpha.0' : '^2.4.0',
+      vuetify: v3 ? '^3.0.0-alpha.5' : '^2.4.0',
```

Comment:
it's going to install lastest alpha anyway so changing to alpha.5 is not really necessary

Reply:
I'm not sure why, but installing dependencies through the CLI doesn't seem take into account version prefixes, there was another occasion where this had to be updated for the fonts as well. 

CLI app package.json:
<img width="355" alt="Screen Shot 2021-06-03 at 2 10 09 PM" src="https://user-images.githubusercontent.com/24883328/120712424-7484a580-c475-11eb-913c-4a2c77baa652.png">

Reply:
> where this had to be updated for the fonts as well

Fonts had a major version bump